### PR TITLE
Fix bugs after self testing

### PR DESCRIPTION
**Pull request by daniwe4**
There is no ticket for this PR. Just fixes that occurs during testing.

### CODE REVIEW

File: setup/check_requirements.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ function check_requirements() {
     if [[ $? -ne 0 ]]
     then
       doil_status_failed
-      doil_status_send_error "REQUIREMENT ERROR" "Please ensure that port 80 and port 443 are available and free!"
+      doil_status_send_error "REQUIREMENT ERROR" "Please ensure that port 80 and port 443 are available and free! Perhaps there is a running Apache!"
```
Comment (tbongers-cat) [inline setup/check_requirements.sh:38]: "... and free! Maybe an Apache webserver is running?"

**Review by tbongers-cat — COMMENTED** [review]
Only a couple ideas for wordings, otherwise it all looks good!

File: README.md [hunk 2]
```diff
@@ -64,6 +64,8 @@ Additional dependencies, but these are installed automatically during setup.
 * psr/log
 * monolog/monolog
 
+The easiest way to fulfill the dependencies is on an Ubuntu 22.01. But it should also be possible with any other Linux installation.
```
[the remark this answers is not in the document]
Comment (daniwe4) [reply README.md]: Done!

File: setup/check_requirements.sh (hunk 1 above)
Comment (daniwe4) [reply setup/check_requirements.sh]: Done!

**Review by tbongers-cat — APPROVED** [review]
lgtm

**Pull request merged**